### DISCUSSION TITLE
Lesson 1 welcome to remix!

### DISCUSSION THREAD

**Discussion by Ugochi-Eze** [smartcontractkit/full-blockchain-solidity-course-py#1535] [General]
Hi Patrick
Was going through the training on creating an application that can store information on the blockchain using solidity, but I don't it, why you use what you use, their meaning. I just don't understand. I also the solidity documentation, went through it a little but I still don't understand.
Please help me to understand. Thank you.

**Comment by cromewar** [comment]
Hello @Ugochi-Eze it was kind of hard to understand your question, could you reformulate please?
